### PR TITLE
media-libs/libmediaart: Drop USE=qt4, fix build w/ Qt 5.7.0

### PR DESCRIPTION
**Pull request by a17r**
Gentoo-bug: 590402

@gentoo/gnome

### CODE REVIEW

**Comment by EvaSDK** [comment]
Why drop the support for qt4 ?

**Comment by a17r** [comment]
It removes a REQUIRED_USE blocker for kde and plasma profile users, and Qt4 is going away anyway. Unless there is any reverse-dep usage for Qt4 (I didn't find any)?

File: media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+		myconf="${myconf} --disable-qt"
+	fi
+
+	gnome2_src_configure \
+		--enable-unit-tests \
+		$(use_enable gtk gdkpixbuf) \
+		$(use_enable introspection) \
+		$(use_enable vala) \
+		${myconf}
+}
```
Comment (SoapGentoo) [inline media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild]: Make this:
```
src_configure() {
	if use qt5; then
		local myconf="--with-qt-version=5"
		append-cxxflags -std=c++11
	fi

	gnome2_src_configure \
		--enable-unit-tests \
		$(use_enable gtk gdkpixbuf) \
		$(use_enable introspection) \
		$(use_enable vala) \
		$(use_enable qt5 qt) \
		${myconf}
}
```

**Comment by kensington** [comment]
Qt 4 is dead and gone, it makes sense to drop it where possible.

File: media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+IUSE="gtk +introspection qt5 vala"
+REQUIRED_USE="
+	?? ( gtk qt5 )
+	vala? ( introspection )
+"
+
+RDEPEND="
+	>=dev-libs/glib-2.38.0:2
+	gtk? ( >=x11-libs/gdk-pixbuf-2.12:2 )
+	introspection? ( >=dev-libs/gobject-introspection-1.30 )
```
Comment (EvaSDK) [inline media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild]: Could you add `:=` as explained here : https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies#Subslot_usage

File: media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+		--enable-unit-tests \
+		$(use_enable gtk gdkpixbuf) \
+		$(use_enable introspection) \
+		$(use_enable qt5 qt) \
+		$(use_enable vala) \
+		${myconf}
+}
+
+src_test() {
+	dbus-launch Xemake check || die #513502
```
Comment (EvaSDK) [inline media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild]: please switch to `virtx emake check` the die becomes unneeded as emake will do it for you.

File: media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+VALA_USE_DEPEND="vapigen"
+VALA_MIN_API_VERSION="0.16"
+
+inherit autotools eutils flag-o-matic gnome2 vala virtualx
```
Comment (EvaSDK) [inline media-libs/libmediaart/libmediaart-1.9.0-r1.ebuild]: Looks like `eutils` is not used anymore since the ebuild uses EAPI 6 so you can drop it.

**Pull request closed without merging**